fast_forward/universe_handler.py

```diff
@@ -36,6 +54,19 @@ def __init__(self, mol_names, *args, **kwargs):
         mol_idxs_names = [(idx, mol_name) for idx, mol_name in self.mol_idxs_by_name.items()]
         self.normal_order = mol_idxs_names.sort()
 
+        istraj = args[1].casefold().endswith(('trr', 'xtc'))
+        
+        # Required for unwrapping
+        if istraj:
+            self.pbc = np.array([ dim2lattice(*f.dimensions) for f in self.trajectory ])
+            self.coordinates = self.trajectory.coordinate_array.astype('float64')
```

Comment:
Do we need float64 accuracy for the PBC treatment? MDAnalysis uses 32 by default and that's also a bit more easy on the memory. Since we keep a lot of stuff there, perhaps using 32 might be the better option.

Reply:
Well, the numpy math (division) seems to cast to float64, which made it much easier to cast everything to float64 (one change) than do everything in float32 (casting in UniverseHandler and casting in the loop in the mapper)

Reply:
that's a valid argument I didn't know numpy implicitly casts them to float64 arrays. Then we leave it like that and just convert them when writing out the universe.